### PR TITLE
Added more front-and-center info about CoreOS Night

### PR DESCRIPTION
**Pull request by ElijahCaine**
@lucywyman thoughts?

### CODE REVIEW

File: content/events/20160211.mkd [hunk 1]
```diff
@@ -1,15 +1,44 @@
...
 preview: Guest Speakers Matthew Garrett and Alex Crawford from CoreOS come to speak for LUG.
-location: Kelly Engineering Center 1001
+location: Kelly Engineering Center 1003
 
 ---
 
 [CoreOS](https://coreos.com/) leaders Matthew Garrett and Alex Crawford will be
 speaking day about Linux, Containers, and more!
 
+## CoreOS Night February 11 @6pm | Kelly Engineering room 1003
```
Comment (lucywyman) [inline content/events/20160211.mkd]: Kelley, it's w/e though if that's the only typo

File: content/home.mkd [hunk 2]
```diff
@@ -10,12 +10,47 @@ pagination:
...
+- - - -
 
 ## The OSU Linux Users Group meets every week
-## _Tuesday at 6:00pm at Kelly Engineering Center in KEC 1007
+## Tuesday at 6:00pm at Kelly Engineering Center in KEC 1007
 
-- - - - 
+- - - -
+
+*Upcomming Event...*
```
Comment (lucywyman) [inline content/home.mkd]: Upcoming

**Comment by lucywyman** [comment]
@ElijahCaine A few typos, otherwise lgtm

**Comment by borcean** [comment]
:+1:

**Pull request merged**